cookbooks/fb_init_sample/recipes/default.rb

```diff
@@ -91,7 +91,7 @@
   node.default['fb_syslog']['_enable_syslog_socket_override'] = false
 end
 include_recipe 'fb_syslog'
-if node.linux? && !node.container?
+if node.fedora_derived? && !node.container?
```

Comment:
Not a node function, a ChefUtils function

```suggestion
if fedora_derived? && !node.container?
```